pycbc/workflow/segment.py

```diff
@@ -850,10 +850,11 @@ def get_triggered_coherent_segment(workflow, out_dir, sciencesegs, tag=None):
                                            'on-before')))
     onafter = int(os.path.basename(cp.get('workflow-exttrig_segments',
                                           'on-after')))
-    padding = int(os.path.basename(cp.get('workflow-exttrig_segments',
-                                          'pad-data')))
+    padding = int(os.path.basename(cp.get('inspiral', 'pad-data')))
```

Comment:
It would be better if the code were not looking into the inspiral section here, as this is then tied to a specific code. Is there another option here? If nothing easy is possible please put a FIXME here.